consent-ui/src/app/ais/password-input-page/password-input-page.component.html

```diff
@@ -1 +1,63 @@
-<p>password-input-page works!</p>
+<div class="d-md-flex align-items-center h-100 align-items-stretch">
```

Comment:
Maybe we could reuse `consent-app-route-based-card-with-sidebar` component for the same styling of all pages

Reply:
yep, done